builders/ethereum/dev-env/thirdweb.md

```diff
@@ -110,17 +112,15 @@ thirdweb offers SDKs for a range of programming languages, such as React, React
 2. Input your preferences for the command line prompts:
 
     1. Give your project a name
-    2. Choose your network. You can choose **EVM** for Moonbeam
-    3. Choose your preferred framework: **Next.js**, **Create React App**, **Vite**, **React Native**, **Node.js**, or **Express**. For this example, you can select **Create React App**
-    4. Choose your preferred language: **JavaScript** or **TypeScript**
+    2. Choose your preferred framework: **Next.js**, **Vite**, or **React Native**. For this example, select **Vite**
 
 3. Use the React or TypeScript SDK to interact with your application’s functions. This will be covered in the following section on interacting with a contract
 
 ## Interact With a Contract {: #interact-with-a-contract }
 
-thirdweb provides several SDKs to allow you to interact with your contract including: [React](https://portal.thirdweb.com/react){target=\_blank}, [React Native](https://portal.thirdweb.com/react-native){target=\_blank}, [TypeScript](https://portal.thirdweb.com/typescript){target=\_blank}, and [Unity](https://portal.thirdweb.com/unity){target=\_blank}.
+thirdweb provides several SDKs to allow you to interact with your contract including: [React](https://portal.thirdweb.com/react/v4){target=\_blank}, [React Native](https://portal.thirdweb.com/typescript/v5/react-native){target=\_blank}, [TypeScript](https://portal.thirdweb.com/typescript/v4){target=\_blank}, and [Unity](https://portal.thirdweb.com/unity){target=\_blank}.
```

Comment:
Why not use react v5 docs?: https://portal.thirdweb.com/typescript/v5/react
and TypeScript v5: https://portal.thirdweb.com/typescript/v5

Reply:
Updated for all except useOwnedNFTs, useContractRead and useContractWrite, which don't have corresponding v5 docs pages.

Reply:
There has to be a v5 alternative, right? I'd prefer to not link back to v4 and just move forward with v5